### PR TITLE
Mac builds

### PR DESCRIPTION
**Pull request by chriddyp**
Brought to you by Travis CI and uploaded to S3.

Unfortunately, doesn't quite work yet, but it's close:
image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - $HOME/.cache/electron-builder
+  yarn: true
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.27.5
+  - export PATH=$HOME/.yarn/bin:$PATH
+
+install:
+  - yarn install
+  - yarn run rebuild:modules:electron
```
Comment (chriddyp) [inline .travis.yml]: @tarzzz - are these still the correct commands?

Comment (tarzzz) [reply .travis.yml]: @chriddyp Yep, these should work.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+script:
+  - yarn run build
+  - yarn run package
```
Comment (tarzzz) [inline .travis.yml]: Just want to make sure that this branch's  `package.js` contains the exception for mac: https://github.com/plotly/falcon-sql-client/blob/master/package.js#L67

Comment (tarzzz) [reply .travis.yml]: This is the relevant issue in case you want to go into details: https://github.com/ibmdb/node-ibm_db/issues/329

**Comment by tarzzz** [comment]
Nice.. working builds.. 💃

**Comment by chriddyp** [comment]
> Nice.. working builds.. 💃

Unfortunately, they don't work yet! Getting this error for some reason, I'm not sure why.
image

**Comment by tarzzz** [comment]
@chriddyp Attempted to fix the failing build + added command to create `dmg` and upload it to S3.. 

I cannot examine builds because I do not seem to have 'download' access to the S3 bucket (Using `Plotly_Editors` account)..

**Comment by chriddyp** [comment]
@tarzzz - Try visiting this link: https://s3.amazonaws.com/falcon-travis-artifacts - You'll see a list of the keys in the bucket and will be able to follow them to get the download link, e.g. https://s3.amazonaws.com/falcon-travis-artifacts/plotly/falcon-sql-client/46/46.1/release.zip

**Comment by tarzzz** [comment]
Update: @cldougl Checked the uploaded binaries with her account, but the builds are still failing with the same error.

**Comment by chriddyp** [comment]
> FWIW, I can download older releases (i.e 48.1 etc.), but not the latest (68.1)

Thanks for reporting! I just modified the S3 policy again, everything should be public-readable now.

**Comment by chriddyp** [comment]
builds are working now! 🎉 
image

File: package.js [hunk 3]
```diff
@@ -59,15 +60,31 @@ function build(cfg) {
...
+                 '`pwd`/node_modules/ibm_db/installer/clidriver/lib/libdb2.dylib ' +
+                 '@loader_path/../../installer/clidriver/lib/libdb2.dylib ' +
+                 'node_modules/ibm_db/build/Release/odbc_bindings.node',
+                 function(err, stdout, stderr) {
+                    console.log(stdout);
+                    if (stderr) {
+                        console.warn('An error occured:' + stderr);
+                        return;
+                    }
+                 });
```
Comment (chriddyp) [inline package.js]: @tarzzz @n-riesco - I still don't understand why we have to do this - I thought that this was done in the ibmdb2 library now? In any case, this was the required piece to make builds work on travis.

Comment (n-riesco) [reply package.js]: @chriddyp https://github.com/ibmdb/node-ibm_db/issues/329 hasn't been fixed in`ibm_db` yet.

**Pull request merged**